index.bs

```diff
@@ -9836,10 +9836,17 @@ Attributes</h5>
 		When an exception is thrown from the processor's
 		<code>constructor</code>, <code>process</code> method,
 		or any user-defined class method, the processor will
-		<a>queue a task</a> on the control thread to fire
-		{{onprocessorerror}} event to the node. Note that
-		once an exception is thrown, the processor will output
-		silence throughout its lifetime.
+		<a>queue a task</a> to fire an
+		<a href="https://www.w3.org/TR/html50/webappapis.html#the-errorevent-interface">
```

Comment:
<https://html.spec.whatwg.org/multipage/webappapis.html#the-errorevent-interface> is better, right?